src/Showcase/Component/Tables.showcase.tsx

```diff
@@ -216,15 +225,15 @@ const Tables = () => {
                         <TableCellDefault size={sizeVirtualizedTable}>{row.calories}</TableCellDefault>
                         <TableCellDefault size={sizeVirtualizedTable}>{row.fat}</TableCellDefault>
                         <TableCellDefault size={sizeVirtualizedTable}>{row.protein}</TableCellDefault>
-                        <TableCellDateTime size={sizeVirtualizedTable} value={row.dateTime}/>
+                        <TableCellDateTime size={sizeVirtualizedTable} value={row.dateTime} />
                       </>
                     )}
                     sortBy={sortBy}
                     sortDirection={sortDirection}
                     endReached={loadMore}
                     overscan={50}
                     components={{
-                      EmptyPlaceholder: undefined
+                      // EmptyPlaceholder: undefined
```

Comment:
Kann man das eigentlich immer noch erzwingen? Sieht nicht so aus... spricht etwas dagegen, das was in `components` (src/Component/Table/VirtualizedTable.tsx Zeile 62) definiert ist einfach durchzureichen?

Reply:
Obsolete da der prop ja reingespreaded wird